partner_firstname/__openerp__.py

```diff
@@ -1,31 +1,21 @@
 # -*- coding: utf-8 -*-
-
-#    Author: Nicolas Bessi. Copyright Camptocamp SA
-#    Copyright (C)
-#       2014:       Agile Business Group (<http://www.agilebg.com>)
-#       2015:       Grupo ESOC <www.grupoesoc.es>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# © 2013 Nicolas Bessi (Camptocamp SA)
+# © 2014 Agile Business Group (<http://www.agilebg.com>)
+# © 2015 Grupo ESOC (<http://www.grupoesoc.es>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Partner first name and last name',
     'summary': "Split first name and last name for non company partners",
-    'version': '8.0.2.0.0',
-    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'version': '9.0.2.0.0',
```

Comment:
You should put 9.0.1.0.0 as is the first version that arrives to 9. We already debated about the convenience of keeping increasing sub-version number and it was decided not to do it.